modules/compute/gke-node-pool/gpu-direct-workload/scripts/enable-tcpx-in-workload.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2024 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import yaml
+import argparse
+
+def main():
+    parser = argparse.ArgumentParser(description="TCPX Job Manifest Generator")
+    parser.add_argument("-f", "--file", required=True, help="Path to your job template YAML file")
```

Comment:
nit: can we add an example for this job template YAML file ?

Reply:
we are not exposing this as module parameter, so there is no easy way for specific testing.
The current gke a3high and a3mega is already testing this script through the sample workload.